releaseNotes/GoFeatureManagement.md

```diff
@@ -2,6 +2,15 @@
 
 [Source code][source_code] | [Package][package] | [Samples][samples]
 
+## v1.0.0 - August 01, 2025
+
+This is the first stable release of Feature Management for Go, including support for the following capabilities. Note that, version `1.1.0-beta.1` or later of Go provider is required for loading feature flags from Azure App Configuration.
+
+- Support for feature flag providers.
```

Comment:
A suggestion:

* Contains built-in support for consuming feature flags defined in Azure App Configuration, as well as an extensibility point via the `FeatureFlagProvider` interface to consume feature flags defined by other providers.
* Supports basic feature flags, as well as feature filters, including:
  * [Microsoft.Targeting](https://github.com/microsoft/FeatureManagement/blob/main/Schema/FeatureFilters/Microsoft.TimeWindow.v1.0.0.schema.json)
  * [Microsoft.TimeWindow](https://github.com/microsoft/FeatureManagement/blob/main/Schema/FeatureFilters/Microsoft.Targeting.v1.0.0.schema.json)
  * Custom filters